IFC4x3/Properties/p/PlasticityIndex_2LRVKFYkv7MBBC_EdSek5O/Documentation.md

```diff
@@ -0,0 +1 @@
+The plasticity index Ip is the numerical difference between the liquid limit and plastic limit of a fine soil.  IP = wL ? wP.
```

Comment:
Not sure.
```suggestion
The plasticity index Ip is the numerical difference between the liquid limit and plastic limit of a fine soil.  IP = wL / wP.
```

Reply:
Should be IP = wL - wP.

Reply:
Fixed in [1315b36](https://github.com/bSI-InfraRoom/IFC-Specification/pull/630/commits/1315b36099fc1e48e7b012240da0b1894a803ab2).